hnginternship5/ziki
# STORY ID [404 page]

#### What does this PR do?
* It creates the 404 page
#### Description of Task to be completed?
* page `404.html` is created
* Styling is added to `inline styling`
#### How should this be manually tested?
* Open the site on your browser
* Type localhost/404
* You should see the landing page
#### Any background context you want to provide?
* use gulp to automate builds
#### What are the relevant pivotal tracker stories?
[#165477920](https://www.pivotaltracker.com/n/projects/2327725)
#### Screenshots
![Screen Shot 2019-04-20 at 11 14 28 AM](https://user-images.githubusercontent.com/38491485/56456058-aeaa9000-635e-11e9-961c-3470d09031e8.png)
Tablet
![Screen Shot 2019-04-20 at 11 14 17 AM](https://user-images.githubusercontent.com/38491485/56456061-ba965200-635e-11e9-8cd2-b35b4fc11ee8.png)
Mobile
#### Questions/Comments:
